### PR TITLE
Handling of potential SecurityException in getting process information.

### PR DESCRIPTION
**Pull request by samartzidis**

### CODE REVIEW

File: Rebus.RabbitMq/RabbitMq/ConnectionManager.cs [hunk 1]
```diff
@@ -259,10 +259,18 @@ static SslOption ToSslOption(SslSettings ssl)
...
+                var currentProcess = Process.GetCurrentProcess(); //This may throw a SecurityException if running from an IIS app. pool process
 
-            properties.Add("ProcessName", currentProcess.ProcessName);
-            properties.Add("FileName", currentProcess.MainModule.FileName);
+                properties.Add("ProcessName", currentProcess.ProcessName);
+                properties.Add("FileName", currentProcess.MainModule.FileName);
+            }
+            catch (System.Security.SecurityException)
+            {
+                // Insufficient permissions to get process information
```
Comment (mookid8000) [inline Rebus.RabbitMq/RabbitMq/ConnectionManager.cs]: wouldn't it be neat to do something like this:

    properties.Add("ProcessName", "Could not be determined.");
    properties.Add("FileName", "Could not be determined.");

?

**Comment by mookid8000** [comment]
cool 👍 I've added a single comment with a question 😄

File: Rebus.RabbitMq/RabbitMq/ConnectionManager.cs (hunk 1 above)
Comment (samartzidis) [reply Rebus.RabbitMq/RabbitMq/ConnectionManager.cs]: Sure that's even better as a quick fix for now. Although I think the ideal solution is to retrieve that info in the way described in the comments but that will need further testing. Thanks for the quick reply!

**Comment by mookid8000** [comment]
it's out in 4.4.1 in a few minutes – thanks 👍

**Pull request merged**